public/scripts/d3/margins.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
+
+var width = 1000;
```

Comment:
Hi, we wanted to give width/heigh etc global scope as it's likely we'll need to refer to it later on in our d3 code – if we use an iife I don't think we'll have access to the variable, right?